### DISCUSSION TITLE
Have you done load test for recommended usage?

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [pocketbase/pocketbase#2083] [Q&A]
In FAQ came to know its best for small & mid size app. But there is no definition for the size, so recommended no of users as well as size data base. Any load test done?

Whats the highest usage had using pocketbase?

**Comment by ganigeorgiev** [comment]
You can check https://github.com/pocketbase/pocketbase/discussions/1629 (_and maybe https://github.com/pocketbase/pocketbase/discussions/417_).

I haven't done recent performance tests, but v0.10+ should yield to higher requests throughput compared to the earlier versions due to the separate read/write-transaction db pools and the auto fail/retry SQLITE_BUSY handling.

Once we reach a little more stable version, I'll consider creating a testbed so that we can start tracking performance improvements/regressions with each release, but for now is a low priority.

If you want to stress test your application, you can use tools like apachebench (ab), k6, bombardier, etc. (_note that the last two are a little more memory intensive_).

**Reply by ganigeorgiev** [reply]
You may also find useful this recent cgo (_sqlite3 col_) vs purego (_sqlite col; the default for PocketBase_) db benchmarks in https://docs.google.com/spreadsheets/d/1YOP1D_ZhuR-ednQhTH6SSrCkcR5bOHzCDoQBZMXvyAw/edit (_note that it has 2 sheets - data and charts_).

**Reply by iamdineshkumar** [reply]
Sure will check that...
